src/Maciek/Fsd.Maciek.Cs/Fsd.Maciek.Cs.Ex2/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Fsd.Maciek.Cs.Ex2.Models;
+
+namespace Fsd.Maciek.Cs.Ex2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Presenter.RenderHeader();
+
+            var AnimalsCollector = new AnimalsCollector();
+            AnimalsCollector.DefineOneOfAnimalType<Cat>("Cat", "Cats");
```

Comment:
to definiowanie mozna wydzielić do osobnej klasy np. AnimalFactory, ale to też jest spoko